package.json

```diff
@@ -79,7 +77,7 @@
     "bundlesize": "~0.17.1"
   },
   "engines": {
-    "node": ">=10.7.0",
+    "node": "8.10.0 || >=10.7.0",
```

Comment:
I think I saw in chat you mentioned that Now v2 only runs Node 8. Is that the reason for this change?

I know @sharkySharks spent a lot of time verifying what version of Node PWA / Venia actually works on; I don't know if `8.10.0` passed the test.

Not sure how to resolve if not, though.

Reply:
I also remember this being the source of a lot of difficulty with `now.sh`. I'm fine with this if it works, though; we can make it clear in the docs that we don't actually support `8.x` if that's the case.

Reply:
Since this is a temporary measure to get a live demo instance working for Imagine I also think this is fine. @jcalcaben do you know if we call out the supported node version anywhere for the server side things?

Reply:
https://magento-research.github.io/pwa-studio/venia-pwa-concept/setup/